docs/book/src/topics/language-specific-examples/azure-identity-sdk.md

```diff
@@ -1,4 +1,11 @@
 # Azure Identity SDK
 
-_Coming soon_
+| Language              | Library                                                               | Minimum Version                                                                                                     |
+| --------------------- | --------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| Go                    | N/A                                                                   | N/A                                                                                                                 |
```

Comment:
I see an [azidentity release](https://github.com/Azure/azure-sdk-for-go/releases/tag/sdk%2Fazidentity%2Fv0.13.0) for go. @udayxhegde Could we use this preview url for the minimum version here?